cmd/kwild/config/flags.go

```diff
@@ -34,16 +34,14 @@ func AddConfigFlags(flagSet *pflag.FlagSet, cfg *KwildConfig) {
 	// Extension endpoints flags
 	flagSet.StringSliceVar(&cfg.AppCfg.ExtensionEndpoints, "app.extension-endpoints", cfg.AppCfg.ExtensionEndpoints, "kwild extension endpoints")
 
-	// TODO: Snapshots are not supported yet
-	// // Snapshot Config flags
-	// flagSet.BoolVar(&cfg.AppCfg.SnapshotConfig.Enabled, "app.snapshots.enabled", cfg.AppCfg.SnapshotConfig.Enabled, "Enable snapshots")
-	// flagSet.Uint64Var(&cfg.AppCfg.SnapshotConfig.RecurringHeight, "app.snapshots.recurring-height", cfg.AppCfg.SnapshotConfig.RecurringHeight, "Recurring snapshot height")
-	// flagSet.Uint64Var(&cfg.AppCfg.SnapshotConfig.MaxSnapshots, "app.snapshots.max-snapshots", cfg.AppCfg.SnapshotConfig.MaxSnapshots, "Maximum snapshots")
-	// flagSet.StringVar(&cfg.AppCfg.SnapshotConfig.SnapshotDir, "app.snapshots.snapshot-dir", cfg.AppCfg.SnapshotConfig.SnapshotDir, "Snapshot directory path")
+	// Snapshot Config flags
+	flagSet.BoolVar(&cfg.AppCfg.Snapshots.Enabled, "app.snapshots.enabled", cfg.AppCfg.Snapshots.Enabled, "Enable snapshots")
+	flagSet.Uint64Var(&cfg.AppCfg.Snapshots.RecurringHeight, "app.snapshots.recurring-height", cfg.AppCfg.Snapshots.RecurringHeight, "Recurring snapshot height")
+	flagSet.Uint64Var(&cfg.AppCfg.Snapshots.MaxSnapshots, "app.snapshots.max-snapshots", cfg.AppCfg.Snapshots.MaxSnapshots, "Maximum snapshots")
```

Comment:
"Maximum snapshots"... to create?  (rotating?)

Reply:
Yeah, we purge old ones and update them with the latest.

Reply:
Sorry I ask a lot of rhetorical questions when commending on docs or help strings.  I'm pretty sure I see the answer, but docs could be tweaked to spell it out to the naive reader.

Reply:
Hahah, That's okay, Thanks for pointing these out, I will improve in my documentation.